client/schema.go

```diff
@@ -49,6 +62,10 @@ func (e ActiveStakingEvent) GetStakingTxHashHex() string {
 	return e.StakingTxHashHex
 }
 
+func (e ActiveStakingEvent) GetEventSchemaVersion() int {
+	return e.SchemaVersion
+}
```

Comment:
One general comment is that why do we need these getters as the fields are already public?